Doolda/Doolda/SignUpScene/SignUpViewModel.swift

```diff
@@ -101,46 +106,38 @@ final class SignUpViewModel: SignUpViewModelProtocol {
     private func bind() {
         self.$emailInput.sink { [weak self] email in
             guard let self = self else { return }
-            self.isEmailValidPublisher.send(self.checkEamilVaild(email))
-            self.checkAllInputVaild()
-        }
-        .store(in: &self.cancellables)
-
-        self.$passwordInput.sink { [weak self] password in
-            guard let self = self else { return }
-            self.isPasswordValidPublisher.send(self.checkPasswordVaild(password))
-            self.isPasswordCheckValidPublisher.send(self.checkPasswordCheckVaild(self.passwordCheckInput))
-            self.checkAllInputVaild()
-
-        }
-        .store(in: &self.cancellables)
-
-        self.$passwordCheckInput.sink { [weak self] passwordCheckInput in
-            guard let self = self else { return }
-            self.isPasswordValidPublisher.send(self.checkPasswordVaild(self.passwordInput))
-            self.isPasswordCheckValidPublisher.send(self.checkPasswordCheckVaild(passwordCheckInput))
-            self.checkAllInputVaild()
+            self.emailValidPublisher = self.checkEamilVaild(email)
```

Comment:
how about using `validateEmail(_ email:)` or `checkEmailValidity(_ email:)`

Reply:
oh.. i agree your opinion. i'll fix it